index.html

```diff
@@ -1,8 +1,118 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
```

Comment:
Make sure to add the javascript line as well once we get to those files.

Reply:
gotcha